data/functions.sql

```diff
@@ -1071,14 +1071,20 @@ DECLARE
   levels_int INTEGER;
   spaces_per_level INTEGER;
 BEGIN
-  -- if the capacity is set, then use that.
-  IF capacity ~ '^[0-9]+$' THEN
+  -- if the capacity is set to something between 0 and 99999, then use that.
+  IF capacity ~ '^[0-9]{1,5}+$' THEN
     RETURN capacity::integer;
   END IF;
+  -- don't try to do this if way_area is abnormally large.
+  -- (Epcot's parking lot is 647k m^2)
+  IF way_area > 700000 THEN
+    RETURN NULL;
+  END IF;
   -- otherwise, try to use the information we have to guess the capacity
   spaces_per_level := (way_area / 46.0)::integer;
   levels_int := CASE
-    WHEN levels ~ '^[0-9]+$' THEN levels::integer
+    -- limit levels to an integer between 0 and 99
+    WHEN levels ~ '^[0-9]{1,3}$' THEN levels::integer
```

Comment:
Isn't this going to allow between 0 and 999?  I thought this second number in the {x, y} was usually inclusive, no?  Same for line 1075

Reply:
Yep. My local code has 2. I must have changed it and not saved/committed.

Reply:
Fixed in 8c8823b.